blockchain/block/blockstore.go

```diff
@@ -12,17 +12,26 @@ import (
 	"github.com/iotexproject/iotex-proto/golang/iotextypes"
 
 	"github.com/iotexproject/iotex-core/action"
-	"github.com/iotexproject/iotex-core/config"
 )
 
 type (
 	// Store defines block storage schema
 	Store struct {
-		Block    *Block
-		Receipts []*action.Receipt
+		EVMNetworkID uint32
+		Block        *Block
+		Receipts     []*action.Receipt
 	}
 )
 
+// NewStore creates a new Store
+func NewStore(evmNetworkID uint32) *Store {
+	return &Store{
+		EVMNetworkID: evmNetworkID,
```

Comment:
I am sorry of my last comment. I was mislead by the *BAD* naming of this struct. 

How could it be named as `store`?!

Reply:
You mean it should be renamed to a more understandable name?

Reply:
let's leave the `Store` as it is. Do not change anything else in `blockdao` and `block`. Let's focus on deleting `config.EVMNetworkID` in this PR.
So, ignore my previous comments, your implementation is fine.